Added Accessibility Testing basics how to article (#1204)
MicrosoftDocs/edge-developer: microsoft-edge/devtools-guide-chromium/accessibility/test-inspect-tool.md

*   **Keyboard-focusable** indicates whether users can reach the element regardless of input device.
    *   A green check mark icon indicates that the element is keyboard-focusable.
    *   A gray circle with diagonal line indicates that the element isn't keyboard-focusable.

## Other information in the Inspect overlay

<!-- general info about the Inspect tool, not particularly focused on accessibility -->

The top part of the **Inspect** overlay in the webpage (above the **Accessibility** section of the overlay) lists applicable details for the element, such as the following:

*   Positioning - if the element is positioned using a flexbox or grid, an appropriate icon \(![Grid layout icon](../media/grid-icon.msft.png)\) is shown.
*   The name of the element, such as `h1`, `h2`, or `div`.
*   The dimensions of the element, in pixels.
*   The color as a color swatch (a small, colored square) and as formatted (such as `#336699`).
*   Font information (size and font families).
*   Margin and padding, in pixels.

> ## Using the Inspect tool for color highlighting of nested regions by hovering over the webpage

Reviewer: ```suggestion
## Identify nested regions using color highlighting 
```

<!-- general info about the Inspect tool, not particularly focused on accessibility -->

> In addition to the information overlay when you hover over the webpage, the **Inspect** tool also provides the same helpful region-coloring behavior as hovering in the DOM tree in the **Elements** tool, but more directly.

Reviewer: ```suggestion
In addition to the information overlay, the **Inspect** tool also provides region-coloring that's similar to hovering in the DOM tree in the **Elements** tool.
```

1.  Open the [accessibility-testing demo webpage][DevToolsA11yErrorsDemopage] in a new tab of the browser, and then select **F12** to open DevTools.

1.  Select the **Inspect** button \(![Inspect tool icon](../media/inspect-icon.msft.png)\) in the top-left corner of DevTools, so that the button is highlighted (blue).

> 1.  Hover over different parts of the rendered demo webpage.  Each element in the webpage now has a multicolor overlay over its entire area, to show any nested regions that the element contains.
> 
> 1.  For example, hover over the left margin of the **Cats** section.  DevTools highlights several rectangular portions of the **Cats** section with different colors, showing the visual outcome of the CSS flexbox definitions.

Reviewer: ```suggestion
1.  Hover over different parts of the rendered demo webpage.  Each element in the webpage now displays with a multicolor overlay. This multicolor overlay can display nested regions inside of an element. For example, hover over the left margin of **Cats**.  DevTools highlights several rectangular portions of **Cats** with different colors, which is the visual outcome of the CSS flexbox definitions on your webpage.
```

> ## Using the Inspect tool to hover over the webpage to highlight the DOM and CSS

Reviewer: ```suggestion
## Use the Inspect tool to hover over the webpage to highlight the DOM and CSS
```

<!-- general info about the Inspect tool, not particularly focused on accessibility -->

1.  Open the [accessibility-testing demo webpage][DevToolsA11yErrorsDemopage] in a new tab of the browser, and then select **F12** to open DevTools.

1.  Select the **Inspect** button \(![the Inspect tool](../media/inspect-icon.msft.png)\) in the top-left corner of DevTools, so that the button is highlighted (blue).

> 1.  Select the **Elements** tool, to make it visible.

Reviewer: ```suggestion
1.  Select the **Elements** tool.
```

1.  With the **Inspect** tool active, hover over different parts of the rendered webpage.  In the **Elements** tool, the HTML DOM tree automatically expands to show information about the element you hover over.  Hovering doesn't cause the **Styles** pane to update.

> 1.  Now select any element within the rendered webpage.  The **Elements** tool automatically opens, showing the HTML of the element in the DOM tree, and also showing the applied CSS for the element in the **Styles** pane.  Selecting an element in the rendered webpage turns off the **Inspect** tool.

Reviewer: ```suggestion
1.  Now select any element within the rendered webpage.  The **Elements** tool automatically opens and displays the HTML of the element in the DOM tree. The tool also displays the applied CSS on the element in the **Styles** pane.  Selecting an element on the rendered webpage turns off the **Inspect** tool.
```
